build.bat

```diff
@@ -0,0 +1,10 @@
+:: Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Given we already have a build.js in the root directory, maybe you can consider give a different name to build.bat and scripts/build.js?

Reply:
working on it.

---
In reply to: [135581459](https://github.com/Microsoft/napajs/pull/47#discussion_r135581459) [](ancestors = 135581459)